content/en/docs/concepts/windows/intro.md

```diff
@@ -352,6 +352,37 @@ Windows Server SAC release
 
 The Kubernetes [version-skew policy](/docs/setup/release/version-skew-policy/) also applies.
 
+## Worker Node Hardware Recommendations {#windows-node-hardware-recommendations}
+
+The minimum hardware requirements for a Windows worker node depend on several
+factors, including the Windows operating system version and specific workload
+requirements. Mentioned below are the minimum system requirements, keeping in
+mind that they can vary based on the components installed on the node and the
+workloads required to run on it.
+
+- 1.4 GHz 64-bit processor, supporting virtualization
+- 8GB or more of RAM
+- NIC card with at least 1 Gbit/s bandwidth
+- At least 20GB of free disk space
```

Comment:
There is at least an authoritative resource to correct this `20 GB`, found on Slack by @aravindhp, that is https://learn.microsoft.com/en-us/windows-server/get-started/hardware-requirements

> Be aware that **32 GB should be considered an absolute minimum** value for successful installation
> (...) A server in Server Core mode is about 4 GB smaller

Reply:
Additionally, some recommendations about disk size can be derived from or point at https://learn.microsoft.com/en-us/azure/aks/cluster-configuration

> default node OS disk configuration uses 128 GB (...)
> The minimum size for AKS images is 30 GB.

It is safe to assume that Windows nodes require the same or at least not less.

Reply:
Thanks @mloskot. Yes given what you shared that value is too low and an ideal minimum would have to be higher, ofcourse depending on the workloads. Will read through those docs to see what relevant information we could also specify here.